src/Drivers/Database.php

```diff
@@ -16,6 +16,15 @@ public function audit(Auditable $model): Audit
     {
         $implementation = Config::get('audit.implementation', \OwenIt\Auditing\Models\Audit::class);
 
+        // If we want to avoid storing Audits with empty old_values & new_values, return null here.
+        if (!Config::get('audit.empty_values')) {
+            $attributeGetter = $model->resolveAttributeGetter($model->getAuditEvent());
+            list($old, $new) = $model->$attributeGetter();
+            if (empty($old) && empty($new)) {
+                return null;
+            }
+        }
```

Comment:
I basically just stole the logic for getting the `$old` and `$new` from `Auditable` and it feels like there's probably a cleaner way to encapsulate this. Open to suggestions.